introduction-to-calmm.md

```diff
@@ -1074,15 +1075,16 @@ Let's then proceed to make an editable list of names.  Here is one way to do it:
 ```jsx
 const ListOfNames = ({names}) =>
   <ul>
-    {U.mapCached(i => <li key={i}><TextInput value={names.view(i)}/></li>,
-                 U.indices(names))}
+    {U.mapElems((name, i) => <li key={i}><TextInput value={name}/></li>,
+                names)}
   </ul>
 ```
 
-Aside from putting the `TextInput` in place, we changed the way elements are
-identified for `U.mapCached`.  In this case we identity them by their index.
-The function `U.indices` from `karet.util` maps a list `[x0, ..., xN]` to a list
-of indices `[0, ..., N]`.  Those indices are then used as the ids.
+Here, we are using
```

Comment:
Obviously feel free to revise as needed. My best guess of what's happening here.